cirq/contrib/acquaintance/gates_test.py

```diff
@@ -290,8 +290,9 @@ def test_get_acquaintance_size():
     op = gate(*qubits[:sum(part_lens)])
     assert cca.get_acquaintance_size(op) == 0
 
-@pytest.mark.parametrize('part_len_sets',
-    [[[randint(1, 5) for _ in range(randint(2, 7))] for _ in range(5)]])
+@pytest.mark.parametrize('part_len_sets', [
+    set(tuple(randint(1, 5) for _ in range(randint(2, 7))) for _ in range(5))
```

Comment:
I ran into this today, found the random sets sometimes were the same. Thanks for fixing.